mmrazor/models/mutables/mutable_channel/__init__.py

```diff
@@ -1,8 +1,31 @@
 # Copyright (c) OpenMMLab. All rights reserved.
+r"""This module defines MutableChannels.
+
+----------------------------------------------------------base_mutable_channel.py
+BaseMutableChannel
+|                            \
+----------------------------------------------------------mutable_channel_container.py
+MutableChannelContainer        \
+----------------------------------------------------------other files
+                                 \   other MutableChannels
+
+MutableChannel are mainly used in DynamicOps. It helps DynamicOps to deal
+with mutable number of channels.
+"""
+from .base_mutable_channel import BaseMutableChannel
```

Comment:
Clean old mutable channels

Reply:
Old MutableChannels are also used by autoslim and slimmable. Remove after refactor?